### PR TITLE
Added method setMinimumSize(int x, int y) : void

### PR DESCRIPTION
**Pull request by Maharal**
When I used the method `setResizable(boolean resizable) : void`, I had trouble resizing the application window. I was able to make windows with 0px height and width, now the programmer can set a minimum window size.

### CODE REVIEW

File: core/src/processing/awt/PSurfaceAWT.java [hunk 1]
```diff
@@ -583,6 +583,15 @@ public void setResizable(boolean resizable) {
     }
   }
 
+    /** Set the minimum size of frame */
+  @Override
+  public void setMinimumSize(int x, int y) {
```
Comment (Shubham-100) [inline core/src/processing/awt/PSurfaceAWT.java]: Use unit instead of int

Comment (Maharal) [reply core/src/processing/awt/PSurfaceAWT.java]: unit? What type is this?

Comment (Shubham-100) [reply core/src/processing/awt/PSurfaceAWT.java]: Oops, I meant uint.

Comment (Maharal) [reply core/src/processing/awt/PSurfaceAWT.java]: I was looking at other methods and none uses the type that you propose. Why should I use the type `uint` you propose? Other methods use int, like: setLocation, setCursor, setSize. And a make a research and I don't find this type.

**Comment by benfry** [comment]
This isn't sufficient to handle sizing across the various renderers and platforms; would need more attention to that.

**Pull request closed without merging**